geode-core/src/main/java/org/apache/geode/cache/wan/GatewaySender.java

```diff
@@ -500,4 +516,12 @@
    */
   void setGatewayEventFilters(List<GatewayEventFilter> filters);
 
+  /**
+   * Set the number of retries to get transaction events
+   * for this GatewaySender when GroupTransactionEvents
+   * is set.
+   *
+   */
+  void setRetriesToGetTransactionEventsFromQueue(int retries);
+
 }
```

Comment:
This method currently is not invoked by anything.

Other setters in this class are invoked by AlterGatewaySenderFunction. Should this property be alterable?

Also, I don't see any tests that test this property.

Reply:
Actually, that method is invoked from `WANTestBase::createSender()` when `groupTransactionEvents` is `true`.
I added it to be able to use a high value for the parameter to avoid flakyness in test cases that have `groupTransactionEvents` set to true. Setting this parameter for test cases using a System Property would not be very convenient.
For uses other than test cases, I think that having the System Property would be enough.